docker-compose.yml

```diff
@@ -13,6 +24,7 @@ services:
       - openwisp_static:/opt/openwisp/static
       - openwisp_media:/opt/openwisp/media
       - openwisp_private_storage:/opt/openwisp/private
+      - openwisp_ssh:/home/openwisp/.ssh
```

Comment:
Is this really necessary?

Reply:
It is the dashboard service that generates SSH key-pair and creates objects in the database.